espnet2/gan_codec/shared/encoder/seanet.py

```diff
@@ -78,6 +78,7 @@ def get_norm_module(
         if causal:
             raise ValueError("GroupNorm doesn't support causal evaluation.")
         assert isinstance(module, nn.modules.conv._ConvNd)
+        norm_kwargs.pop("num_groups", None)
```

Comment:
Not pretty sure, does it have any side-effect?

Reply:
I modified this part. It should be good now.